_data/join.yml

```diff
@@ -63,3 +63,8 @@
   required: false
   type: "String"
   description: "Set to “true” to indicate that the user is a guest, otherwise do NOT send this parameter."
+
+- name: "role"
+  required: false
+  type: "String"
+  description: "Define user role for the meeting. Accept the values(case insensitive) MODERATOR or VIEWER. If the role parameter is present and it's a valid option, it will take over of any password parameter provided."
```

Comment:
```suggestion
  description: "Define user role for the meeting. Accept the values(case insensitive) MODERATOR or VIEWER. If the role parameter is present and it's a valid option, it will take over of any password parameter provided. Added in BBB 2.4"
```